lib/connections/limiter_test.go

```diff
@@ -169,8 +169,7 @@ func TestAddAndRemove(t *testing.T) {
 	checkActualAndExpected(t, actualR, actualW, expectedR, expectedW)
 }
 
-func initConfig(t *testing.T) {
-	t.Helper()
+func initConfig() {
```

Comment:
This can just return a config, rather than setting some global that leaks between the tests.